### ISSUE TITLE
LOSC to GWOSC

### ISSUE THREAD

**Issue by Cyberface** [gwastro/pycbc#2375]
Not sure if the names of files have changed on https://www.gw-openscience.org/ but I was just looking around code and noticed the `frame/losc.py` module. Should this be updated? 

Also a quick search on github for `losc` gives the following

https://github.com/gwastro/pycbc/search?q=losc&unscoped_q=losc

Should these be updated too?

**Comment by ahnitz** [comment]
@Cyberface Yes, those should be updated too. So far, I had just done the minimal to keep things working.
